packages/api/models/Clin.ts

```diff
@@ -0,0 +1,8 @@
+export interface Clin {
+  clin_number: string;
+  idiq_clin: string;
+  total_clin_value: number;
+  obligated_funds: number;
+  pop_start_date: string;
+  pop_end_date: string;
```

Comment:
It feels like it might be a little awkward to have to repeat the word `clin` so often. `clin.clin_number` and `clin.total_clin_value` and so on` since it's very likely that this is either going to be accessed either as a variable named clin or as an attribute where we've referenced `.clins[]` or something like that. Perhaps `number`, `total_value`? I am unsure about `idiq_clin`. Is this going to require an API change?

We should consider adding TSDoc for the interface(s) and their members too (across the board).

Reply:
This comes from an unnamed object that is a collection of named properties directly from the API spec, so I used the exact names as specified.

```
          clins:
            type: array
            items:
              type: object
              properties:
                clin_number:
                  type: string
                idiq_clin:
                  type: string
                total_clin_value:
                  type: "integer"
                obligated_funds:
                  type: "integer"
                pop_start_date:
                  type: string
                  format: date
                  example: '2021-07-01'
                pop_end_date:
                  type: string
                  format: date
                  example: '2022-07-01'
```

Reply:
Perhaps code reads awkwardly.  Let's discuss outside the PR whether that is deserving of a named schema object Clin, and whether names should be adjusted.

Reply:
@jeffsegal-ccpo See the above.

Reply:
Not a bad idea to have a named object but we should maybe go back to make sure we're 100% on the functional requirements, e.g. the relationship between the "parent" CLIN and the IDIQ CLIN that it maps to.